src/coreclr/src/jit/gentree.h

```diff
@@ -5991,23 +5991,35 @@ struct GenTreePhiArg : public GenTreeLclVarCommon
 
 struct GenTreePutArgStk : public GenTreeUnOp
 {
+private:
+    unsigned m_byteOffset;
+#ifdef FEATURE_PUT_STRUCT_ARG_STK
+    unsigned m_byteSize; // The number of bytes that this argument is occupying on the stack with padding.
```

Comment:
I gather that this doesn't put this node above the small node size limit?

Reply:
They are already large nodes:
https://github.com/dotnet/runtime/blob/72b7d236ad634c2280c73499ebfc2b594995ec06/src/coreclr/src/jit/gentree.cpp#L273-L280